services/backend/src/core/profile/profile.js

```diff
@@ -406,7 +408,7 @@ async function updateProfile(request, response) {
                   create: i,
                   update: {
                     level: i.level,
-                    date: normalizeDate(i.date, "day"),
+                    date: i.date && normalizeDate(i.date, "day"),
```

Comment:
This case is already handled in normalizeDate (for the null and undefined value of its first param, `i.date` in this case)

Reply:
removed unneeded code